### DISCUSSION TITLE
Documentation for Microsoft.WinForms.Designer.SDK

### DISCUSSION THREAD

**Discussion by ovonick** [dotnet/winforms#6602] [Q&A]
Hello WinForms team.

Is there documentation available for the Microsoft.WinForms.Designer.SDK? If not, is there an ETA for it?
Nuget [page](https://www.nuget.org/packages/Microsoft.WinForms.Designer.SDK/) links to a nonexistent github repo.

**Comment by tbolon** [comment]
I *think* something is expected for the following days/weeks, certainly on the [visual studio blog](https://devblogs.microsoft.com/dotnet/state-of-the-windows-forms-designer-for-net-applications) regarding the winforms designer and the new SDK.

There are already [some people](https://github.com/dotnet/winforms/discussions/5220#discussioncomment-1974061) who have experienced using the new designer SDK without documentation.

You can explore the SDK by yourself, or watch the dotnet blog for new articles from @KlausLoeffelmann.

**Reply by KlausLoeffelmann** [reply]
Yes. I am on it! 😊
